static/script/devices/mediaplayer/live/seekable.js

```diff
@@ -76,9 +76,11 @@ require.def(
                 this._mediaPlayer.playFrom(offset);
             },
 
-            pause: function () {
+            pause: function (disableAutoResume) {
```

Comment:
I feel like this might be better as an opts object, as `pause(true)` or `pause(false)` isn't very readable. I would prefer to see something like:
```
var opts = {
    autoResume: false
};
pause(opts);
```
Also, I feel like autoResume should be off by default rather than on, but that might just be a personal thing.